archiver.rb

```diff
@@ -0,0 +1,32 @@
+require "tempfile"
+require "fileutils"
+require "find"
+
+class Archiver
+  attr_accessor :file
+  @@modes = [:reader, :writer, :both]
+
+  def initialize
```

Comment:
Since you are not passing any initial value to the class instance you could get rid of this initialize(constructor method) without any issue

Reply:
Got it! , I erased the initialize function and the modules, and closed the file. TY for your review and your suggestions!